src/helpers/library.ts

```diff
@@ -297,17 +297,17 @@ type RecentGame = {
   title: string
 }
 
+const Store = window.require('electron-store')
+const configStore: ElectronStore = new Store({
+  cwd: 'store'
+})
+
 function getRecentGames(library: GameInfo[]) {
-  return library.filter((game) => {
-    const Store = window.require('electron-store')
-    const configStore: ElectronStore = new Store({
-      cwd: 'store'
-    })
-    const recentGames: Array<RecentGame> =
-      (configStore.get('games.recent') as Array<RecentGame>) || []
-    const recentGamesList = recentGames.map((a) => a.appName) as string[]
-    return recentGamesList.includes(game.app_name)
-  })
+  const recentGames =
+    (configStore.get('games.recent') as Array<RecentGame>) || []
+  const recentGamesList = recentGames.map((a) => a.appName) as string[]
+
+  return library.filter((game) => recentGamesList.includes(game.app_name))
```

Comment:
While coding this feature initially I used a similar method like this one so I included this refactor. The refactor is a small improvement (we don't need to initialize the store and query it for every callback function call) but I can remove it from this PR if it adds noise, since it's not really needed for this feature at the end.